config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -29,7 +29,8 @@ spec:
     logLevel: info
     enableKubeProbes: true
     healthPort: 8080
-    prometheusPort: 9090
+    prometheusPort: 9102
+    metricsConfigFile: 'config/flp/flowlogs-pipeline.conf.yaml'
```

Comment:
please embed in the code like what was suggested by @ronensc in the team slack channel